meta/inszenium/isotope-export/en.yml

```diff
@@ -0,0 +1,7 @@
+en:
+    title: Isotope eCommerce Export
+    description: >
+        The extension set new options for export orders.
```

Comment:
What about `This extension provides new options for order exports.`? It doesn't really `set` anything.